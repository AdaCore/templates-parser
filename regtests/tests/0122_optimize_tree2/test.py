from test_support import *

run('print_tree', ["-M", "macro.tmplt"])
run('testme', ["macro.tmplt"])
