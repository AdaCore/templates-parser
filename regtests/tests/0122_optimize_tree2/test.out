[INFO] macro.tmplt
[C_INFO]  1 FALSE
[TEXT] --
[TEXT] /ME/
--
/ME/
